docs/cosmos-hub/genesis-state.md

```diff
@@ -0,0 +1,60 @@
+# Gaia Genesis State
```

Comment:
what's the difference between this file and `docs/cosmos-hub/genesis.md` ? if this one is Gaia specific, I guess we can move the other to the SDK docs and link it from here